interface.md

```diff
@@ -198,32 +198,34 @@ Not all variants need to be supported – for example, a posterior model might n
 model.
 
 
-### Density Calculation
+### Density Evaluation
 
 Since the different “versions” of how a model is to be understood as generative or conditioned are
 to be expressed in the type or dispatch they support, there should be no need for separate functions
-`logjoint`, `loglikelihood`, etc., which force these semantic distinctions on the implementor; one
-`logdensity` should suffice for all, with the distinction being made by the capabilities of the
-concrete model instance.
+`logjoint`, `loglikelihood`, etc., which force these semantic distinctions on the implementor; we
+therefore adapt the interface of
+[DensityInterface.jl](https://github.com/JuliaMath/DensityInterface.jl).  Its one function
+`logdensityof` should suffice for variants, with the distinction being made by the capabilities of
+the concrete model instance.
```

Comment:
It's actually two functions (or three, if one includes `logfuncdensity` which seems irrelevant here though), maybe mention `densityof` as well?